src/run.ts

```diff
@@ -24,16 +25,17 @@ export type Outputs = {
   pullRequestNumber?: number
 }
 
-export const run = async (inputs: Inputs): Promise<Outputs> => {
+export const run = async (inputs: Inputs, context: Context, octokit: Octokit): Promise<Outputs> => {
   if ((await git.status()) === '') {
     core.info('Nothing to commit')
     return {}
   }
 
   await git.configureAuthor()
 
-  if (github.context.eventName === 'pull_request') {
-    return await handlePullRequestEvent(inputs, github.context as PullRequestContext)
+  if (contextIsPullRequestEvent(context)) {
```

Comment:
This resolves https://github.com/int128/update-generated-files-action/issues/864